### PR TITLE
extended 64-bit monotonic timer

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: examples/rtic-tick.rs [hunk 1]
```diff
@@ -20,8 +23,10 @@ mod app {
         led: PC13<Output>,
     }
 
-    #[monotonic(binds = TIM2, default = true)]
-    type MicrosecMono = MonoTimerUs<pac::TIM2>;
+    //#[monotonic(binds = TIM2, default = true)]
+    //type MicrosecMono = MonoTimerUs<pac::TIM2>;
```
Comment (therealprof) [inline examples/rtic-tick.rs:27]: Seems duplicate?

File: examples/rtic-tick.rs [hunk 2]
```diff
@@ -30,8 +35,10 @@ mod app {
 
         let gpioc = ctx.device.GPIOC.split();
         let led = gpioc.pc13.into_push_pull_output();
+        defmt::info!("Start");
 
-        let mono = ctx.device.TIM2.monotonic_us(&clocks);
+        //let mono = ctx.device.TIM2.monotonic_us(&clocks);
```
Comment (therealprof) [inline examples/rtic-tick.rs:40]: Same here.

File: examples/rtic-tick.rs (hunk 1 above)
Comment (burrbull) [reply examples/rtic-tick.rs:27]: no, it is not

Comment (therealprof) [reply examples/rtic-tick.rs:27]: I mean, what is the purpose of having a commented block of 2 lines that is identical to the 2 lines before?

Comment (burrbull) [reply examples/rtic-tick.rs:27]: Those "2 lines before" are deleted lines. Red.

**Review by therealprof — APPROVED** [review]
bors r+

**Pull request merged**